server/views/partials/js-head.njk

```diff
@@ -1,30 +1,65 @@
+      A.prototype.load=function(a,b){var c=this,k=a||"BESbswy",q=0,C=b||3E3,F=(new Date).getTime();return new Promise(function(a,b){null===E&&(E=!!document.fonts);if(E){var J=new Promise(function(a,b){function e(){(new Date).getTime()-F>=C?b():document.fonts.load(I(c,'"'+c.family+'"'),k).then(function(c){1<=c.length?a():setTimeout(e,25)},function(){b()})}e()}),K=new Promise(function(a,c){q=setTimeout(c,C)});Promise.race([K,J]).then(function(){clearTimeout(q);a(c)},function(){b(c)})}else m(function(){function t(){var b;
+      if(b=-1!=f&&-1!=g||-1!=f&&-1!=h||-1!=g&&-1!=h)(b=f!=g&&f!=h&&g!=h)||(null===B&&(b=/AppleWebKit\/([0-9]+)(?:\.([0-9]+))/.exec(window.navigator.userAgent),B=!!b&&(536>parseInt(b[1],10)||536===parseInt(b[1],10)&&11>=parseInt(b[2],10))),b=B&&(f==u&&g==u&&h==u||f==v&&g==v&&h==v||f==w&&g==w&&h==w)),b=!b;b&&(null!==d.parentNode&&d.parentNode.removeChild(d),clearTimeout(q),a(c))}function G(){if((new Date).getTime()-F>=C)null!==d.parentNode&&d.parentNode.removeChild(d),b(c);else{var a=document.hidden;if(!0===
+      a||void 0===a)f=e.a.offsetWidth,g=n.a.offsetWidth,h=p.a.offsetWidth,t();q=setTimeout(G,50)}}var e=new r(k),n=new r(k),p=new r(k),f=-1,g=-1,h=-1,u=-1,v=-1,w=-1,d=document.createElement("div");d.dir="ltr";x(e,I(c,"sans-serif"));x(n,I(c,"serif"));x(p,I(c,"monospace"));d.appendChild(e.a);d.appendChild(n.a);d.appendChild(p.a);document.body.appendChild(d);u=e.a.offsetWidth;v=n.a.offsetWidth;w=p.a.offsetWidth;G();z(e,function(a){f=a;t()});x(e,I(c,'"'+c.family+'",sans-serif'));z(n,function(a){g=a;t()});x(n,
+      I(c,'"'+c.family+'",serif'));z(p,function(a){h=a;t()});x(p,I(c,'"'+c.family+'",monospace'))})})};"undefined"!==typeof module?module.exports=A:(window.FontFaceObserver=A,window.FontFaceObserver.prototype.load=A.prototype.load);}());
+
+      var rootElement = document.documentElement;
+      var enhancedClass = 'enhanced';
+      var enhancedScriptPath = '/assets/js/app.js';
+
+      var WBSubset = new FontFaceObserver('Wellcome Bold Web Subset', {weight: 'bold'});
+      var WB = new FontFaceObserver('Wellcome Bold Web', {weight: 'bold'});
+      var HNL = new FontFaceObserver('Helvetica Neue Light Web');
+      var HNM = new FontFaceObserver('Helvetica Neue Medium Web');
+      var LR = new FontFaceObserver('Lettera Regular Web');
+
+      function addClass() {
+        if (!rootElement.classList.contains(enhancedClass)) {
+          rootElement.className += ' ' + enhancedClass;
```

Comment:
Only problem is I'm calling addClass twice, so we can end up with the class twice (although that probably doesn't matter)

Reason being, addClass is called optimistically at the same time the javascript file is called. If the javascript file hasn't loaded after 8 seconds, the class gets removed, so we don't end up with enhanced styles, but no enhanced functionality.

However, there is no way I know of to cancel the javascript request, so if it does end up loading we need to add the class again, hence the second addClass call.